scapy/sendrecv.py

```diff
@@ -783,6 +786,19 @@ def _run(self,
                     tcpdump(fname, args=["-w", "-", flt], getfd=True)
                 ), label) for fname, label in six.iteritems(offline))
             else:
+                # Write Scapy Packet objects to a pcap file
+                def _write_to_pcap(packets_list):
+                    fd, filename = tempfile.mkstemp(suffix=".pcap")
```

Comment:
I wonder if we don't want to use `scapy.utils.get_temp_file(autoext=".pcap")` instead

Reply:
You are right!